docqueries/dijkstra/doc-pgr_dijkstra.result

```diff
@@ -686,8 +686,8 @@ SELECT * FROM pgr_dijkstra(
    2 |        2 |         2 |       3 |    3 |   -1 |    0 |        1
 (2 rows)
 
--- q18
--- q19
+/* -- q18 */
+/* -- q19 */
```

Comment:
same here

Reply:
I have added two blank lines manually and pushed  new iteration.